bedrock/firefox/templates/firefox/whatsnew/whatsnew-fx68-b.html

```diff
@@ -0,0 +1,164 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% add_lang_files "firefox/whatsnew_67.0.5" %}
+
+{% extends "firefox/whatsnew/whatsnew-fx68-trailhead.html" %}
+
+{% block page_css %}
+  {{ css_bundle('firefox_whatsnew_modular') }}
+{% endblock %}
+
+{% block body_attrs %}{{ super() }} data-variant="b"{% endblock %}
+
+{% block content %}
+<main class="mzp-t-firefox content-wrapper {% block container_class %}{% endblock %}">
+  <header class="c-page-header">
+    <div class="mzp-l-content c-page-header-inner">
+    {% block header_logo %}
+      {{ high_res_img('logos/firefox/logo-quantum-wordmark-large.png', {'alt': 'Firefox', 'width': '122', 'height': '40', 'class': 'c-page-header-logo-fx'}) }}
+    {% endblock %}
+      <h1 class="c-page-header-up-to-date"><span>{{ _('Congrats! You’re using the latest version of Firefox.') }}</span></h1>
+    </div>
+  </header>
+
+  <section class="content-main">
+    <div class="mzp-l-content">
+
+      <div class="c-emphasis-box show-fxa-default show-fxa-supported-signed-out">
+      {% block main_content_signed_out %}
+        <h2 class="c-emphasis-box-title">{{ _('You’ve got the latest browser. Now get the rest of Firefox.') }}</h2>
+        <p>{{ _('Join Firefox for a whole family of respectful products, plus all the knowledge you need to protect yourself online.') }}</p>
+        <div class="c-signup-container fxa-form-cta">
+          {% block fxa_form %}
+            {{ fxa_email_form(
+                entrypoint='mozilla.org-whatsnew6705',
+                button_class='mzp-c-button mzp-t-product mzp-t-small',
+                utm_source='whatsnew6705',
+                style='trailhead',
+                utm_campaign='trailhead',
+                utm_content='get-the-rest-of-firefox')
```

Comment:
I think in this case the `utm_content` should be the experiment variant, something like `whatsnew68-experiment-b`?

Reply:
alternatively we could even update `utm_source` and `entrypoint` to be `mozilla.org-whatsnew68-b`?
[best practices](https://ga-dev-tools.appspot.com/campaign-url-builder/) seems to suggest using `utm_content` for A/B testing, though.